src/main/java/uk/gov/companieshouse/api/accounts/validation/RptTransactionValidator.java

```diff
@@ -12,6 +12,8 @@
 public class RptTransactionValidator extends BaseValidator {
 
     private static final String TRANSACTION_BREAKDOWN_PATH_BALANCE_AT_PERIOD_START = "$.transaction.breakdown.balance_at_period_start";
+    private static final String[] TRANSACTION_TYPE_MESSAGE = {"money given to the company by a related party", "money given to a related party by the company"};
```

Comment:
Why use an array here? Wouldn't an Map be better and use the .containsValue method to check if the correct value or an Enum and use the EnumUtils to check?
Have we decided to send the whole message from the web or a key that is stored?

Reply:
I believe with FRS101/102 we send the string as a whole to the API from the WEB. As for using array i'd say it does seem strange but would work fine, could even simply it and take them out of the array and just have them as 2 constants? See what everyone else says 👍